mobile/lib/modules/sharing/views/album_viewer_page.dart

```diff
@@ -143,7 +138,8 @@ class AlbumViewerPage extends HookConsumerWidget {
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
             _buildTitle(albumInfo),
-            _buildAlbumDateRange(albumInfo),
+            if (albumInfo.assets != null && albumInfo.assets!.isNotEmpty)
```

Comment:
However, is there any syntactic sugar that can further improve the conditional statement inside the build method, just wondering?

Reply:
Oh good point!
It can be change like this
 'if(albumInfoassets?.isNotEmpty == - true)'
I'll change it come back home

Reply:
I found more.